WISE-Community/WISE-Client
# Grade By Team does not load if a component has an annotation and then the component is deleted

How to test
1. Create a run (or use an existing run)
2. Have a student save work for component1
3. Have the teacher give a score to that student on component1
4. Delete component1
5. Open the Classroom Monitor for the run
6. Go to the "Grade By Team" page. The "Grade By Team" view should load. Before the fix, the view used to not load and then display the error below in the console.

```
angular.js:15570 TypeError: Cannot read property 'excludeFromTotalScore' of null
    at TeacherProjectService.shouldIncludeInTotalScore (projectService.ts:2444)
    at annotationService.ts:299
    at Array.filter (<anonymous>)
    at AnnotationService.getScoreAnnotations (annotationService.ts:296)
    at AnnotationService.getTotalScore (annotationService.ts:283)
    at TeacherDataService.getTotalScoreByWorkgroupId (teacherDataService.ts:744)
    at StudentProgressController.getStudentTotalScore (studentProgressController.ts:108)
    at StudentProgressController.updateTeam (studentProgressController.ts:135)
    at StudentProgressController.initializeStudents (studentProgressController.ts:127)
    at new StudentProgressController (studentProgressController.ts:50)
```

Closes #249
